sparkles/core.py

```diff
@@ -106,39 +107,58 @@ def run_aca_review(load_name=None, *, acas=None, make_html=True, outdir=None,
     "all" which generates a report for every obsid.
 
     :param load_name: name of loads
-    :param acas: list of ACAReviewTable objects (optional)
+    :param acars: list of ACAReviewTable objects (optional)
     :param make_html: make HTML output report
-    :param outdir: output directory
+    :param report_dir: output directory
     :param report_level: report level threshold for generating acq and guide report
     :param roll_level: level threshold for suggesting alternate rolls
     :param loud: print status information during checking
     :param obsids: list of obsids for selecting a subset for review (mostly for debug)
     :param is_ORs: list of is_OR values (for roll options review page)
+    :param raise_exc: if False then catch exception and return traceback (default=True)
+    :returns: exception message: str or None
 
     """
-    if acas is None:
-        acas = get_acas_from_pickle(load_name, loud)
+    try:
+        _run_aca_review(load_name=load_name, acars=acars, make_html=make_html,
+                        report_dir=report_dir, report_level=report_level,
+                        roll_level=roll_level, loud=loud, obsids=obsids)
+    except Exception:
```

Comment:
@jeanconn - the exception is caught here.

Reply:
Thanks!  That's all I needed.  So for interface, @jskrist just needs to change both the `get_aca_catalog` call to add `raise_exc=False` and the `acar.run_aca_review(raise_exc=False)` to get full protection and not have confusing stuff at console.

Reply:
Done and done.  Now if I could only get roll_option messages without generating an html file: #73